frontend/README.md

```diff
@@ -0,0 +1,29 @@
+# Create T3 App
```

Comment:
Similarly, I would expect a single `README` file at the root, where the removed one is. This one seems to be generated, so we can probably just delete it. I think there's already an issue to correct a proper readme later anyways.